README.md

```diff
@@ -1,71 +1,36 @@
-# External Secrets
-
-<img src="assets/round_eso_logo.png" width="100">
+<p align="center">
+    <img src="assets/eso-logo-large.png" width="30%" align="center" alt="external-secrets">
+</p>
 
-----
-
-The External Secrets Operator reads information from a third party service
-like [AWS Secrets Manager](https://aws.amazon.com/secrets-manager/) and automatically injects the values as [Kubernetes Secrets](https://kubernetes.io/docs/concepts/configuration/secret/).
+# External Secrets
+![ci](https://github.com/external-secrets/external-secrets/actions/workflows/ci.yml/badge.svg?branch=main)
+[![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/5327/badge)](https://bestpractices.coreinfrastructure.org/projects/5947)
+[![Go Report Card](https://goreportcard.com/badge/github.com/external-secrets/external-secrets)](https://goreportcard.com/report/github.com/external-secrets/external-secrets)
+<a href="https://artifacthub.io/packages/search?repo=external-secrets"><img alt="Artifact Hub" src="https://img.shields.io/endpoint?url=https://artifacthub.io/badge/repository/external-secrets" /></a>
```

Comment:
It seems this search link is broken? Should it be `https://artifacthub.io/packages/helm/external-secrets-operator/external-secrets`?

Can we use https://shields.io/ to also create a badge for operatorhub?